ui-bootstrap/js/components/content.tsx

```diff
@@ -39,6 +39,26 @@ interface IContentState {
     isImageUploadOpen: boolean;
 }
 
+export const checkCharacterLength = (keyCode: string, maxSize: number, text: string): boolean => {
+    const validKeyCodes = [
+        'Backspace',
+        'Delete',
+        'ArrowUp',
+        'ArrowDown',
+        'ArrowLeft',
+        'ArrowRight',
+    ];
+
+    const count: number = text.length;
+    if (count >= maxSize) {
+        if (!(validKeyCodes.indexOf(keyCode) > -1)) {
```

Comment:
these ifs could be combined with &&
`if (count >= maxSize && !(validKeyCodes.indexOf(keyCode) > -1)) {`
at which point you could return if condition itself, if you think that looks ok